src/style.css

```
@@ -1,3 +1,75 @@
+/** {*/
+/*  box-sizing: border-box;*/
+/*}*/
+
 body {
+  --text-hover-color: #00acdc;
+  font-family: "Roboto", sans-serif;
   margin: 0;
 }
+
+.header {
+  display: flex;
+  min-width: 1024px;
+  justify-content: space-between;
+}
+
+.nav {
+  margin-right: 50px;
+}
+
+.logo__img {
+  padding-top: 10px;
+  padding-left: 50px;
+}
```

You can rewrite this
```suggestion
.header {
  display: flex;
  width: 100$;
  justify-content: space-between;
  align-items: center;
  height: 60px;
}
```